aiohttp/web_main.py

```diff
@@ -0,0 +1,43 @@
+from .web import run_app
+from argparse import ArgumentParser
+from importlib import import_module
+
+
+def main():
+    arg_parser = ArgumentParser(
+        description="aiohttp.web TCP/IP Application server",
```

Comment:
`HTTP` is redundant; `aiohttp.web` already implies it's `HTTP`.
I'll make it `aiohttp.web Application server` in case the `TCP/IP` part is too clunky.